src/app/ApplicationTemplate.UWP/Views/Content/Diagnostics/DiagnosticsOverlay.xaml

```diff
@@ -29,20 +30,27 @@
 				<Setter Property="Foreground"
 						Value="WhiteSmoke" />
 			</Style>
+
+			<uc:FromNullableBoolToCustomValueConverter x:Key="TrueToHorizontalStretch"
+													   TrueValue="Stretch"
+													   NullOrFalseValue="Right" />
 		</ResourceDictionary>
 	</UserControl.Resources>
 
 	<StackPanel Background="#AA000000"
 				BorderThickness="1"
-				BorderBrush="#000000">
+				BorderBrush="#000000"
+				HorizontalAlignment="{Binding IsDiagnosticsExpanded, Converter={StaticResource TrueToHorizontalStretch}, FallbackValue=Right}"
+				VerticalAlignment="Top">
```

Comment:
Should we stretch it vertically too?

Reply:
Sure. But now, my question is do we want to fully expand or keep a certain margin (look at the screenshot in the PR I've updated it). I'm asking, because, at the moment, the overlay has a top margin and I'm wondering if we should keep that on expanded.